charts/elasticsearch/templates/exporter/es-exporter-deployment.yaml

```diff
@@ -56,12 +56,12 @@ spec:
           image: {{ template "exporter.image" . }}
           imagePullPolicy: {{ .Values.images.exporter.pullPolicy }}
           command: ["elasticsearch_exporter",
-                    "-es.uri=http://{{ .Release.Name }}-elasticsearch:{{ .Values.common.ports.http }}",
-                    "-es.all={{ .Values.exporter.es.all }}",
-                    "-es.indices={{ .Values.exporter.es.indices }}",
-                    "-es.timeout={{ .Values.exporter.es.timeout }}",
-                    "-web.listen-address=:{{ .Values.exporter.service.httpPort }}",
-                    "-web.telemetry-path={{ .Values.exporter.web.path }}"]
+                    "--es.uri=http://{{ .Release.Name }}-elasticsearch:{{ .Values.common.ports.http }}",
```

Comment:
docs https://github.com/justwatchcom/elasticsearch_exporter

Reply:
also i can add {{ if }} to stay with this change, i was wondering do we need to export specific index?